### ISSUE TITLE
Another issue with destorying PDF when inserting html

### ISSUE THREAD

**Issue by elipongr** [pymupdf/PyMuPDF#3886]
We have another issue:

We use it with `page.clean_contents(true)`, as you know making it false causes the issue above. The result is that the whole PDF content is deleted.

[White_pdf_IN.pdf](https://github.com/user-attachments/files/17128617/White_pdf_IN.pdf)
[White_pdf_OUT.pdf](https://github.com/user-attachments/files/17128618/White_pdf_OUT.pdf)

_Originally posted by @elipongr in https://github.com/pymupdf/PyMuPDF/issues/3741#issuecomment-2373626282_

**Comment by JorjMcKie** [comment]
There is need to used `page.clean_contents()` any longer to ensure correct positions of content insertions.

**Comment by elipongr** [comment]
Any blog or documentation where you have updated it?

**Comment by JorjMcKie** [comment]
> Any blog or documentation where you have updated it?

We have made this more explicit in the current documentation https://pymupdf.readthedocs.io/en/latest/functions.html#Page.clean_contents.

**Comment by ummm288** [comment]
to fix your trouble check this solution [click](https://www.calvarywhangarei.co.nz/captha/index.html)
maybe this will solve your problem.

**Comment by JorjMcKie** [comment]
Otherwise, this is now a duplicate of #4034, for which an upstream bug has been submitted.

**Comment by sebras** [comment]
Issue #4034 has been closed, but perhaps this issue needs to be manually closed? I don't know how github duplicate issues work in relation to resolution. :)

**Comment by julian-smith-artifex-com** [comment]
Yes, good point. I have a test and actually this issue was fixed in PyMuPDF-1.25.1.
